docs/source/index.rst

```diff
@@ -11,69 +11,17 @@ JupyterLab is the next-generation web-based user interface for Project Jupyter.
 
 .. image:: ./images/jupyterlab.png
    :align: center
-   :class: jp-screenshot
+   :class: jp-screenshot shadow
 
+Get started
+===========
 
+See the sections below (and the top-level links above) for more information about using, extending, and contributing to JupyterLab.
 
 .. toctree::
-   :maxdepth: 1
-   :caption: Getting Started
+   :maxdepth: 2
```

Comment:
I would add a caption because the default one is _Get Started_ and I found it confusing as it does not match the _Get Started_ from the navigation top bar.

```suggestion
   :maxdepth: 2
   :caption: Table of Contents
```